src/main/java/cz/muni/fi/pv286/mmt/exceptions/InvalidArrayBracketException.java

```diff
@@ -0,0 +1,16 @@
+package cz.muni.fi.pv286.mmt.exceptions;
+
+import java.io.IOException;
+
+/**
+ * An exception that is thrown when an error related to brackets occurs.
+ */
+public class InvalidArrayBracketException extends IOException {
+    public InvalidArrayBracketException(String message) {
+        super(message);
+    }
```

Comment:
**suggestion**: Remove unused function.

It's a code smell and the code it's harder to read.